examples/io/plot_read_cfradial2.py

```diff
@@ -0,0 +1,28 @@
+"""
+==========================================================
+Read and Plot Cfradial2/FM301 data Using Xradar and Py-ART
+==========================================================
+
+An example which uses xradar and Py-ART to read and plot Cfradial2/FM301 data.
+
+"""
+
+# Author: Max Grover (mgrover@anl.gov)
+# License: BSD 3 clause
+
+
+import xradar as xd
+
+import pyart
+from pyart.testing import get_test_data
+
+# Locate the test data and read in using xradar
+filename = get_test_data("cfrad2.20080604_002217_000_SPOL_v36_SUR.nc")
+tree = xd.io.open_cfradial1_datatree(filename)
```

Comment:
Should this be cfradial2? Or does xradar handle both for that datatree?

Reply:
I fixed it in the example - we can use xarray open datatree directly!